packages/lib-user/src/components/GroupStats/components/CopyIcon/CopyIcon.js

```diff
@@ -0,0 +1,22 @@
+import { Blank } from 'grommet-icons'
+
+function CopyIcon(props) {
```

Comment:
Just to double check if this icon needs attribution? There is a grommet-icons Copy icon that could be used if this CopyIcon svg was accidentally just copy-pasted from somewhere.

Reply:
I'm not sure about attribution, same as the Add icon the current path's are per Figma.
Also not sure if can use grommet-icon instead.

For reference, here's the Figma based icon:
![Copy icon per Figma](https://github.com/zooniverse/front-end-monorepo/assets/12118751/b6d504d4-a4ab-4f98-881f-f9e4914de560)

And here are some grommet-icon options:
![Grommet copy related icons](https://github.com/zooniverse/front-end-monorepo/assets/12118751/0cc11c6b-89d5-4259-8642-2a966cb7808d)

Reply:
@seanmiller26 where did the icons in Figma originate from for these? The "Add" icon and "Copy" icons in lib-user are currently just copy-pasted svg code and without attribution. Can `grommet-icons` be used instead?

Reply:
Yes, we should always use Grommet icons. Either Clone or Layer will work here.